src/OVAL/probes/unix/linux/partition_probe.c

```diff
@@ -402,7 +402,7 @@ int partition_probe_main(probe_ctx *ctx, void *probe_arg)
                                 rc = oscap_pcre_exec(re, mnt_entp->mnt_dir,
                                                strlen(mnt_entp->mnt_dir), 0, 0, NULL, 0);
 
-                                if (rc == 0) {
+                                if (rc >= 0) {
```

Comment:
What do you think, would it make sense to use `if (rc > OSCAP_PCRE_ERR_NOMATCH)` is such cases?